src/Helper/Service/BasePathFactory.php

```diff
@@ -4,32 +4,27 @@
 
 namespace Laminas\View\Helper\Service;
 
-use ArrayAccess;
 use Laminas\View\Helper\BasePath;
 use Psr\Container\ContainerInterface;
 
 use function assert;
-use function is_array;
 use function is_string;
 
 final class BasePathFactory
 {
+    /** @psalm-suppress MixedAssignment */
```

Comment:
Eh, let's suppress per-statement, not just everywhere? Mixed assignments are very sneaky 

Fine to dismiss this review comment, if you believe it's not as important.